### DISCUSSION TITLE
Developing bot

### DISCUSSION THREAD

**Discussion by Zainab963** [community/community#135605] [Programming Help]
### Body

Can the Github community provide guidance on developing bots?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by UjjwalSaini07** [comment]
which type of developing bot u want

**Comment by UjjwalSaini07** [comment]
Yes, the GitHub community can provide guidance on developing bots. You can explore GitHub Discussions, join community forums, or check out repositories related to bot development. Additionally, you can ask questions or seek advice on platforms like Stack Overflow or GitHub Issues in relevant repositories. GitHub also has documentation and guides that cover building and integrating bots using GitHub Actions, the GitHub API, and other tools.

**Comment by Zainab963** [comment]
Hi Ujj.. thanks for replying. I need to build a bot that can automate
cataloguing and classification activities/services in a library. The bot
should also be able to teach and examine students on practical cataloguing
and classification. Is that possible? I need to build it myself. I am new
to this though. I am a librarian by profession

**Reply by UjjwalSaini07** [reply]
U know any language

**Comment by Zainab963** [comment]
Thanks Ujjwal. I would like to develop a bot to manage cataloguing
processes in a library.
